### ISSUE TITLE
 Debug mode on server side (CLI) ?

### ISSUE THREAD

**Issue by serge-hulne** [neutralinojs/neutralinojs#277]
I think a debug mode on the command line is needed to debug errors which are not related to the JS scripts (i.e. which are not "client-side"), but to anything else: e.g. : Configuration scripts (json files), for instance to avoid. 

For instance : a typo in config.json will make the executable fail silently (at least on Windows 10).

it could be passed as an option to the executable, lilke neutralino-win.exe --debug, for instance.

That's what a regular server does by default (Express, for instance).

**Comment by shalithasuranga** [comment]
Yeah this could be a nice feature for developers. I am thinking to display log file changes continuously with `neu run`
